cli/tests/e2e/snapshots/test_ci/test_nosem/noautofix---enable-nosem/output.txt

```diff
@@ -22,6 +22,10 @@ Findings:
           ⋮┆----------------------------------------
          11┆ y == y
           ⋮┆----------------------------------------
+         23┆ b == b # Triage ignored by syntactic_id
```

Comment:
shouldn't these not show up here since they are ignored?

Reply:
No, this test does not use the CI for rules, it uses the registry. So if there's no response from the app then there's no issues to triage ignore
No login token:
https://github.com/returntocorp/semgrep/blob/7e330b3f29b942a03237262e94d668037fcdf9a6/cli/tests/e2e/test_ci.py#L758-L766